widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/extensions/ViewExtensions.kt

```diff
@@ -395,3 +396,13 @@ internal fun Context.wrapWithMaterialThemeOverlay(
         defStyleRes
     )
 }
+
+internal fun ViewGroup.hasChildOfType(clazz: Class<*>): Boolean {
+    for (i in 0 until childCount) {
```

Comment:
could be `children.any{ clazz.isInstance(it) }`

Reply:
@DavDo You are smarter than ChatGPT 😄 💪